packages/webapp/src/views/developer/email-entry.html

```diff
@@ -0,0 +1,130 @@
+                            name: "emailAddresses["+index+"]",
+                            spellcheck: false,
+                            autocomplete: "email",
+                            value: email,
+                            errorMessage: getErrById(err, fieldId),
+                            label: {
+                                text: "Email address",
+                                classes: "govuk-label"
+                            }
+                        }) }}
+                        {{ govukButton({
+                            type: "button",
+                            text: 'Remove',
+                            classes: 'govuk-button govuk-button--secondary moj-add-another__remove-button custom-remove-btn'
+                        }) }}
+                        {% endcall %}
+                        {% endif %}
+                        {% set index = index + 1 %}
+                    {% endfor %}
+                </div>
+                {% endif %}
+
+                <div data-module="moj-add-another">
+                    {% call govukFieldset({
+                        classes: 'moj-add-another__item',
+                        legend: {
+                        text: 'Email',
+                        classes: 'moj-add-another__title govuk-fieldset__legend--m',
+                        isPageHeading: false
+                        }
+                    }) %}
+                
+                    {{ govukInput({
+                        id: "emailAddresses["+emailIndex+"]",
+                        name: "emailAddresses[]",
+                        autocomplete: "email",
+                        spellcheck: false,
+                        value: emailAddresses[emailIndex],
+                        errorMessage: getErrById(err, "emailAddresses["+emailIndex+"]"),
+                        label: {
+                            text: "Email address",
+                            classes: "govuk-label"
+                        },
+                        attributes: {
+                            "data-name": "emailAddresses[%index%]",
+                            "data-id": "emailAddresses[%index%]"
+                        }
+                    }) }}
+                    {% endcall %}
+
+                    <div class="moj-button-action">
+                        {{ govukButton({
+                            type: "button",
+                            text: 'Add another email address',
+                            classes: 'govuk-button--secondary moj-add-another__add-button govuk-!-margin-bottom-4'
+                        }) }}
+                    </div>
+                </div>
+                {{ govukButton({
+                    text: "Continue",
+                    attributes: {
+                        id: "continue"
+                    }
+                }) }}
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block bodyEnd %}
+    {{ super() }}
+    <script>
```

Comment:
Any scripting needs to go into its own script file, please see other pages with client side scripting